docs/instrumentation-list.yaml

```diff
@@ -4681,34 +4608,115 @@ libraries:
           type: BOOLEAN
   jaxws:
   - name: jaxws-2.0
+    display_name: JAX-WS
+    description: This instrumentation enables controller spans for JAX-WS Provider
+      implementations.
+    library_link: https://github.com/jakartaee/jax-ws-api
+    features:
+    - CONTROLLER_SPANS
+    - CONTEXT_PROPAGATION
+    - EXPERIMENTAL_ONLY
     source_path: instrumentation/jaxws/jaxws-2.0
     scope:
       name: io.opentelemetry.jaxws-2.0
     target_versions:
       javaagent:
       - javax.xml.ws:jaxws-api:[2.0,]
+    configurations:
+    - name: otel.instrumentation.common.experimental.controller-telemetry.enabled
+      description: Enables the creation of experimental controller (INTERNAL) spans.
+      type: boolean
+      default: false
+    telemetry:
+    - when: otel.instrumentation.common.experimental.controller-telemetry.enabled=true
+      spans:
+      - span_kind: INTERNAL
+        attributes:
+        - name: code.function
+          type: STRING
+        - name: code.namespace
+          type: STRING
   - name: jaxws-2.0-axis2-1.6
+    display_name: Apache Axis2 1.6 JAX-WS 2.x
+    description: |
+      This instrumentation enables controller spans for Apache Axis2 JAX-WS web services, and enriches HTTP server spans with route information.
+    library_link: https://axis.apache.org/axis2/java/core/
+    features:
+    - CONTROLLER_SPANS
+    - CONTEXT_PROPAGATION
```

Comment:
does it really do context propagation?

Reply:
https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/instrumentation/jaxws/jaxws-2.0-axis2-1.6/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/axis2/Axis2Helper.java#L21-L53